### PR TITLE
BIRT

### PR DESCRIPTION
**Pull request by frosario**

### CODE REVIEW

File: birt.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+docker-compose -f compose/birt.yml up -d birt
+
+##Setup up the settings json file
+#echo '{"public": {"analyticsSettings": {"Google Analytics" : {"trackingId": "CHANGE-ME"} } } }' > settings-production.json
+#docker exec -t birt mkdir /shared
+#docker cp settings-production.json birt:/shared/settings-production.json
+
+#Restart birt
+#docker kill birt
+#docker start birt
```
Comment (arnoldbm) [inline birt.sh:51]: Did you mean to comment this out?

File: compose/birt.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+birt:
+  container_name: birt
+  image: birt
+  environment:
+    - MONGO_URL=mongodb://10.0.0.205:27017/birt
+    - MONGO_URI=mongodb://10.0.0.205:27017
+    - MONGO_HOST=10.0.0.205
+    - MONGO_DB=birt
+    - MONGO_PORT=27017
+    - SIMULATION_PORT=45000
```
Comment (arnoldbm) [inline compose/birt.yml]: SIMULATION_PORT is only applicable for FLIRT

File: birt.sh (hunk 1 above)
Comment (frosario) [reply birt.sh:51]: I did actually. I wasn't sure if the forced restart was necessary for this particular app. I commented it out so I could easily revert back if I was wrong.

File: compose/birt.yml (hunk 2 above)
Comment (frosario) [reply compose/birt.yml]: I'll strip that var out then

**Comment by arnoldbm** [comment]
Looks good!

**Pull request merged**